cmd/wallet/address.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
```

Comment:
Please revert this file. we don't need specify flag on importing address

Reply:
<img width="1001" alt="Screenshot 2024-11-05 at 10 38 56 AM" src="https://github.com/user-attachments/assets/3b55bbfa-e870-48de-a778-26b3b9f5ea6d">
Please try to import and export private key for testnet address.

Reply:
Could you please explain to me how to resolve this issue?

Reply:
How did it happen?  Explain how to reproduce it...